### PR TITLE
[Homework #7] - JS-Part 2 by Oleksii Achkevych

### PR DESCRIPTION
**Pull request by ste1nerr**

### CODE REVIEW

**Comment by ste1nerr** [comment]
what is the right way for me to do that while I have the first pr not merged, I need to create a new branch and a new pr and that it does not include items from the previous one

**Comment by ste1nerr** [comment]
okay, I get it, my mistake was that I was splitting off from the branch of the previous homework, and I should have been splitting off from the main

File: homeworks/oleksii-achkevych/07-js-basics-2/script.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+// 1. https://www.codewars.com/kata/5715eaedb436cf5606000381
+function positiveSum(array) {
+    let sum = 0;
+    for (let i = 0; i < array.length; i++) {
+      if (array[i] > 0)
+        sum += array[i];
+    }
+    return sum;
+  }
```
Comment (ilya-ischenko) [inline homeworks/oleksii-achkevych/07-js-basics-2/script.js:9]: Reduce method?

**Review by ilya-ischenko — COMMENTED** [review]
For some reason I see that you are trying to push 6 and 7 hw, fix it, please

**Pull request closed without merging**